docs/spaces/index.asciidoc

```diff
@@ -2,28 +2,112 @@
 [[xpack-spaces]]
 == Spaces
 
-With spaces, you can organize your dashboards and other saved objects into meaningful categories.
-After creating your own spaces, you will be asked to choose a space when you enter Kibana. 
-Once inside a space, you will only see the dashboards and other saved objects that belong to that space. 
+Spaces enable you to organize your dashboards and other saved 
+objects into meaningful categories. Once inside a space, you see only 
+the dashboards and saved objects that belong to that space. 
 
-You can change your current space at any time, by clicking on the space avatar in the top left.
+{kib} creates a default space for you. 
+After you create your own 
+spaces, you're asked to choose a space when you enter Kibana. You can change your 
+current space at any time.
 
 [role="screenshot"]
 image::spaces/images/change-space.png["Change current space"]
 
-With security enabled, you can <<xpack-security-authorization, control which users have access to individual spaces>>.
+Kibana supports spaces in several ways.  You can:
 
+[[spaces-getting-started]]
+
+* <<spaces-managing, View&comma; create&comma; and delete spaces>>
```

Comment:
I'm not sure if there is a way to control this or not, but the tooltips here all have the word "edit" suffixed at the end:

![spaces docs - link tookips](https://user-images.githubusercontent.com/3493255/61535602-acf57480-aa00-11e9-9c00-13c294a6923b.gif)

Reply:
@legrego Thanks for the review.  I'm don't see the tooltip error.  What browser are you using?

Reply:
hmm, I'm using Chrome 75.0.3770.100 on MacOS, and I ran the `docbldkbx --open` command to generate the docs

Reply:
You've found https://github.com/elastic/docs/issues/641. It's never been super high on my list to be honest.

Reply:
Thanks for the clarification @nik9000!